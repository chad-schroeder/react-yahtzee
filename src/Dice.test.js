import React from 'react';
import { shallow } from 'enzyme';
import Dice from './Dice';
import toJson from 'enzyme-to-json';

it('renders without crashing', () => {
  shallow(<Dice dice={[undefined]} locked={[false]} />);
});

it('matches snapshot', () => {
  let wrapper = shallow(
    <Dice dice={[undefined, undefined]} locked={[true, false]} />
  );
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});
