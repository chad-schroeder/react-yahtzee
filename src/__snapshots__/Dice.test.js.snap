// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="Dice"
>
  <Die
    idx={0}
    key="0"
    locked={true}
  />
  <Die
    idx={1}
    key="1"
    locked={false}
  />
</div>
`;
